clusterloader2/testing/load/config.yaml

```diff
@@ -77,7 +77,6 @@
 {{$schedulerThroughputPodsPerDeployment := .Nodes}}
```

Comment:
Whole scheduler-throughput section could be moved to the module. We do not plan to modify those parameters.
The end goal is to simplify main config as much as possible.

Reply:
So I kept this one due to the 'load' section that starts on the line 94, which also uses these parameters, , in case we want to move it to the beginning of the scheduler-throughput model, do we want to leave something at around the line 94 where the config starts or it's not necessary and so after variable declarations, config will start with Starting measurements?

Reply:
I think that creating ($namespace + 10) namespaces  in large clusters would solve that issue, at the cost of not explicit interdependence. Let's leave this for now as it is.